runtime/seq.go

```diff
@@ -173,6 +173,24 @@ func seqFindFirst(f *Frame, iterable *Object, pred func(*Object) (bool, *BaseExc
 	return false, nil
 }
 
+func seqFindElem(f *Frame, elems []*Object, o *Object, start, stop int) (int, bool, *BaseException) {
```

Comment:
Have the return value be `(int, *BaseException)` and have the first return value be -1 when not found.